### PR TITLE
expose system clock on T0/PB0

### PR DESCRIPTION
**Pull request by pavelskipenes**
[Datasheet](datasheet%2FMicrocontroller%20ATmega162.pdf) references Clock Output Buffer on page 40.
image

To get the output it required two preconditions. 
- `PB0` is set as output    
- `CKOUT` bit is set (active low) in fuse low byte (page 233 / Table 100)

the latter can be done by `avrdude -c atmelice -p m162 -U flash:w:0xbd:w` which is now in the make rules for flashing.

### CODE REVIEW

**Comment by pavelskipenes** [comment]
waiting for #10 before merge into main

**Review by KristianKristian — APPROVED** [review]
very clocky

**Pull request merged**
